README.md

```diff
@@ -1,24 +1,70 @@
-# README
+# RE-FORMER
+> In this project, i have built a basic a form from scratch  using Ruby on Rails
```

Comment:
- [ ] **OPTIONAL**
It'd be very nice if you attach a screenshot of your cool design in the README file. 🙏 😄